e2e/tests/upgrades/upgrade_test.go

```diff
@@ -272,7 +272,8 @@ func (s *UpgradeTestSuite) TestV5ToV6ChainUpgrade() {
 	var hostAccount string
 
 	t.Run("register interchain account", func(t *testing.T) {
-		version := getICAVersion(testconfig.GetChainATag(), testconfig.GetChainBTag())
+		// explicitly set the version string because intertx with ibfc-go v5 does not support incentivized channels.
+		version := icatypes.NewDefaultMetadataString(ibctesting.FirstConnectionID, ibctesting.FirstConnectionID)
```

Comment:
I set here also the default version for the reason stated in the comment above this line.